microsoft-edge/extensions-chromium/store-policies/developer-policies.md

```diff
@@ -315,13 +315,13 @@ If your Extension provides content \(such as user-generated, retail, or other we
 
 ### 2.13 Videos  
 
-If you submit a promotional video in the listing, it should follow all the content guidelines mentioned in this policy.  If you choose to provide a YouTube link, you must ensure that advertisements are disabled for the specific videos that you want to embed.  For more information about how ads are enabled and disabled on YouTube, see [https://support.google.com/youtube/answer/2531367?ref_topic=7072227][GoogleYoutubeAnswer2531367Topic7072227] and [https://support.google.com/youtube/answer/132596][GoogleYoutubeAnswer132596].  
+If you submit a promotional video in the listing, it should follow all the content guidelines mentioned in this policy.  If you choose to provide a YouTube link, you must ensure that advertisements are disabled for the specific videos that you want to embed.  For more information about how ads are enabled and disabled on YouTube, see [support.google.com/youtube/answer/2531367?ref_topic=7072227][GoogleYoutubeAnswer2531367Topic7072227] and [support.google.com/youtube/answer/132596][GoogleYoutubeAnswer132596].  
 
 <!-- image links -->  
 
 <!-- links -->  
 
-[MicrosoftEdgeContentSecurityPolicyRemoteScript]: csp.md#relaxing-the-default-policy "Relaxing the default policy - Content Security Policy \(CSP\) "  
+[MicrosoftEdgeContentSecurityPolicyRemoteScript]: ./csp.md#relaxing-the-default-policy "Relaxing the default policy - Content Security Policy \(CSP\) | Microsoft Docs"  
```

Comment:
What is this change being made for? I would need some context.

Reply:
The change was made to bring the link into better consistency with documentation guidelines.